sceptre/plan/actions.py

```diff
@@ -41,6 +41,7 @@ class StackActions(object):
     def __init__(self, stack):
         self.stack = stack
         self.name = self.stack.name
+        self.rel_path = self.stack.rel_path
```

Comment:
It's not good practice to abbreviate variable names. It only saves a few keystrokes and sacrifices clear understanding. A better variable name would be useful for communicating intent.